File: Beecrowd/1150/1150.c
#include <stdio.h>

int main(){

    int x, z;
    scanf("%d", &x);
    while(1){
        scanf("%d", &z);
        if(z > x){
            break;
        }
    }
    int sum = 0, count = 0;
    for(int i = x; sum <= z; i++){
        sum += i;
        count++;
    }
    printf("%d\n", count);

    return 0;
}

File: Beecrowd/1150/1150.py
# Problem statement: https://www.beecrowd.com.br/judge/en/problems/view/1150
x = int(input())
while True:
    z = int(input())
    if z > x:
        break
sum = 0
count = 0
for i in range(x, z+1):
    if sum > z:
        break
    sum += i
    count +=1
print(count)

